logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -1030,9 +1047,11 @@ class UserSessionScope internal constructor(
 
     internal val mlsMigrationWorker get() =
         MLSMigrationWorkerImpl(
-            mlsMigrationRepository,
+            userConfigRepository,
+            featureConfigRepository,
+            MLSConfigHandler(userConfigRepository, updateSupportedProtocolsAndResolveOneOnOnes, userId),
```

Comment:
suggestion: isn't better to define these as variables then we can re-use them?

Reply:
Define the config handlers as separate variables and re-use them you mean?

Reply:
Done